src/content/getting-started/architecture/route-agent/_index.en.md

```diff
@@ -1,17 +1,21 @@
-+++
-title =  "Route Agent"
-weight = 30
-+++
+---
+title: "Route Agent"
+---
 
-The Route Agent component runs on every node in each participating cluster. It is responsible for setting up VXLAN tunnels and routing the
-cross cluster traffic from the node to the cluster’s active Gateway Engine which subsequently sends the traffic to the destination cluster.
+The _Route Agent_ runs on every node in each connected cluster. It is
+responsible for setting up the necessary host network elements on top
+of the existing network plugin.
 
-When running on the same node as the active Gateway Engine, Route Agent creates a VXLAN VTEP interface to which Route Agent instances
-running on the other worker nodes in the local cluster connect by establishing a VXLAN tunnel with the VTEP of the active Gateway Engine
-node. The MTU of the VXLAN tunnel is configured based on the MTU of the default interface on the host minus the VXLAN overhead.
+For **_kubeproxy-iptables_** based implementations it creates VxLAN tunnels from
+the node to the cluster's active Gateway node, as well as routing
+to allow normal Pod and host-network connectivity to remote clusters.
+When a Gateway fails and HA failover happens the route-agent handler will update
+the VxLAN tunnels as well as the routes.
 
-Route Agents use `Endpoint` resources synced from other clusters to configure routes and to program the necessary iptables rules to enable
-full cross-cluster connectivity.
+For **_OVNKubernetes_** host-network routing is configured on all nodes, and
+on the active Gateway node forwarding is configured between the **_ovn-k8s-gw0_**
+and the cable interface.
 
-When the active Gateway Engine fails and a new Gateway Engine takes over, Route Agents will automatically update the route tables on each
-node to point to the new active Gateway Engine node.
```

Comment:
A lot of this detail has been removed.  I assume it still applies (for kubeproxy-iptables anyway) in which case I think we should still retain it and work in the OVNKubernetes details around it.

Reply:
Ack, I have restored that part.